pyccl/tracers.py

```diff
@@ -434,8 +420,9 @@ def _get_MG_transfer_function(self, cosmo, z):
 
         return mg_transfer
 
+    @warn_api
```

Comment:
In `star` I had regrouped them because `is_logt` refers to the transfer function. Up to you if you want to do it.

Reply:
I saw that. I think I prefer this order (and it doesn't matter much, since now they're all keyword args.